### PR TITLE
Restore missing localizations for Linux AppImage

### PR DESCRIPTION
**Pull request by shoogle**
Resolves: #1382

1. Ensure INSTALL_PREFIX is set correctly.
    - The CMake variable INSTALL_PREFIX is now correctly passed into the program.
2. Make Audacity portable.
    - Audacity now checks for resources at a location relative to the executable instead of a fixed location each time.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -332,7 +332,7 @@ else()
 endif()
 
 set( _DEST "${_DESTDIR}" )
-set( _PREFIX "${CMAKE_INSTALL_PREFIX}" )
+set( INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}" )
```
Comment (shoogle) [inline CMakeLists.txt:335]: Needed for [audacity_config.h.in](https://github.com/audacity/audacity/blob/c6ef82360a66d6fde071b546a44b566df4fadc3f/src/audacity_config.h.in#L36).

Comment (Paul-Licameli) [reply CMakeLists.txt:335]: Understood:  the .in file doesn't change but was assuming assignment of a Cmake variable had happened, when it had not.

File: src/CMakeLists.txt [hunk 2]
```diff
@@ -1089,7 +1089,6 @@ list( APPEND LIBRARIES
 )
 
 set( BUILDING_AUDACITY YES )
-set( INSTALL_PREFIX "${_PREFIX}" )
```
Comment (Paul-Licameli) [inline src/CMakeLists.txt:1092]: So this assignment is lifted to a higher scope and done earlier.  Got it.

File: src/AudacityApp.cpp [hunk 3]
```diff
@@ -1206,28 +1216,23 @@ bool AudacityApp::OnInit()
...
       audacityPathList);
-   FileNames::AddUniquePathToPathList(wxString::Format(wxT("%s/share/doc/audacity"),
-      wxT(INSTALL_PREFIX)),
+   FileNames::AddUniquePathToPathList(installPrefix + L"/share/doc/audacity",
       audacityPathList);
 #endif //AUDACITY_NAME
 
-   FileNames::AddUniquePathToPathList(wxString::Format(wxT("%s/share/locale"),
-      wxT(INSTALL_PREFIX)),
+   FileNames::AddUniquePathToPathList(installPrefix + L"/share/locale",
```
Comment (Paul-Licameli) [inline src/AudacityApp.cpp:1235]: LGTM

**Review by Paul-Licameli — APPROVED** [review]

**Comment by Penikov** [comment]
Works fine with appimage

изображение

**Pull request merged**